### ISSUE TITLE
Swept Force Close Missing?

### ISSUE THREAD

**Issue by 555-FILK** [MutinyWallet/mutiny-web#379]
7/13/2023 @ 0950: 100,000 sats lightning received, 90,000 sats channel open.
7/13/2023 @ 1112: channel force closed because I opened the wallet on another device.
7/13/2023 @ 1130: 100,000 sats lightning received, 90,000 sats channel open.
7/13/2023 @ 1305: channel force closed because I opened the wallet on another device.
7/14/2023 @ 1755: 88,940 sats on-chain receive Swept Force Close.
7/14/2023 @ 1846: 88,940 sats channel open.

That second forced channel close of 90,000 sats on 7/13 @ 1305 have not been returned to me. I thought by now they would be. Should I keep waiting or is something wrong?
![IMG_5294](https://github.com/MutinyWallet/mutiny-web/assets/80064814/9020fb30-1f72-4bee-9fd4-d28a7b9ca1e1)

**Comment by TonyGiorgio** [comment]
Please quit opening the wallet on multiple devices at the same time. It was not designed for this and it is why you lost funds here. 

Please provide logs, your node id (at the top of admin tools in the settings) and any transaction id's you can so we can investigate where the funds ended up to see if we can recover them for you.

**Comment by 555-FILK** [comment]
Ya ya ya I know now :) .

Node ID: 0295ff34eaf4c4a44b5c814dcfb79c1c9806bd753b6c706b64f400e4f132ba1dc0

7/13/2023 @ 0950: sats received = lnbc900u1pjtq2h5dqqnp4q22l7d827nz2gj6us9xulduurjvqd0t48dk8q6my7sqwfufjhgwuqpp5ctnlcxudayxgqqt0j4mzvajy0sthf8zgwsweehrasrlmgxvdlh3ssp5vf2k2hjpfud34cac762kfydm4tml40jelttnpne2da0jf5ujuj2q9qyysgqcqzptxqrpwu3g4lwfpp3nvqp0pfk6hzaq5a22t9n2650jujvpj9t4hcqf7yfmwswzupcf9lstteg9h8nhyrjsrl8hsal3xzhh93lrcahasypqw240gqh94ect

7/13/2023 @ 1130: sats received = lnbc900u1pjtqs5zdqqnp4q22l7d827nz2gj6us9xulduurjvqd0t48dk8q6my7sqwfufjhgwuqpp549pnxsxu4yr4tk550cvv88qu59a3am3lekkakd2k6aj7t52p0mqqsp5n4mcunqmrzpx74643p6q4rd6cn7zlngpgh7u5khqt704f6fzw9lq9qyysgqcqzptxqrpwujw4kfa0jwn9l4rvemsedcxn022yywl4zq4m2h6nfuah4h4zclsvjxa36fp38ku58d44wgcf2eapnqw5p6y6xf45tjc4lpfa8d43kz6gqrq3a7q

7/14/2023 @ 1755: swept force closed = 343aaa91730e3bfac8b5f0c86fb240ae890241cb2dc787e4c2c922a3b605962d

7/14/2023 @ 1846: channel open = 9348118c9ce247e2fc11b6cb240db3e6e62e98f75b699db953a97865e7bfe81d

[mutiny-logs.txt](https://github.com/MutinyWallet/mutiny-web/files/12089904/mutiny-logs.txt)

**Comment by TonyGiorgio** [comment]
Thanks. I have forwarded this along to the LSP.

**Comment by 555-FILK** [comment]
Thanks legend.

**Comment by TonyGiorgio** [comment]
Can you check your channels in the settings? I think 2 other channels should currently be opened to you with some balance on your side, in addition to that last one you opened for 88k sats.

**Comment by 555-FILK** [comment]
Yes I have 3 total open channels.

**Comment by TonyGiorgio** [comment]
Is that expected? Are those working fine? Just trying to figure out the accounting here and how much is expected on your side. The funding transaction for one of the channels does not exist which is why there's that channel close notification.

**Comment by 555-FILK** [comment]
Yes and working fine. Just passing around sats with friends on Mutiny.
![IMG_5298](https://github.com/MutinyWallet/mutiny-web/assets/80064814/552253ac-d6ce-49a0-aa77-37e8dd2a8b1d)
![IMG_5299](https://github.com/MutinyWallet/mutiny-web/assets/80064814/146994c4-83af-40b4-b835-4c0a715d19c2)

**Comment by TonyGiorgio** [comment]
We figured out what happened to it, do you have an on chain address we can return the funds to? Feel free to DM me or the mutiny account on nostr or twitter if you want to keep it private.

**Comment by 555-FILK** [comment]
bc1qc9lahncrfmuhapsxgh69uqftfun3j3895l32jg

Thank you.

**Comment by SeanGriffithz** [comment]
Hi, the same thing happened to me... I jumped the gun and opened the app on multiple devices... and 90,000 are stuck too (coincidentally).

**Comment by TonyGiorgio** [comment]
> Hi, the same thing happened to me... I jumped the gun and opened the app on multiple devices... and 90,000 are stuck too (coincidentally).

please open a new ticket, provide logs, your node id, and any relevant transactions and details.

**Comment by TonyGiorgio** [comment]
> bc1qc9lahncrfmuhapsxgh69uqftfun3j3895l32jg
> 
> Thank you.

Sent!

**Comment by 555-FILK** [comment]
Received, thank you.

What do I do about the 90,000 sats on-chain balance the wallet shows that is not actually there?

**Comment by TonyGiorgio** [comment]
I think another user just ran into that, try the resync button in the admin tools setting page https://github.com/MutinyWallet/mutiny-web/issues/382#issuecomment-1644252145

**Comment by 555-FILK** [comment]
No dice.

If I "delete everything" and re-load the wallet with my seed phrase, will everyone die?
